src/ducks/apps/components/Sections.jsx

```diff
@@ -105,4 +109,4 @@ export class Sections extends Component {
   }
 }
 
-export default translate()(withBreakpoints()(Sections))
+export default translate()(withBreakpoints()(withRouter(Sections)))
```

Comment:
You should also add some propTypes. Always a good idea ;)

Reply:
Yep, I am currently trying to get back this habit to add propTypes :)